src/main/resources/XWiki/SuggestDisplay.xml

```diff
@@ -453,6 +516,31 @@ ul.picker-list .picker-item {
     #end
     ## now display the item in edit mode, to have the suggest and all.
     $doc.displayEdit($propertyClass, $prefix, $object)
+
+    ## Save the className and the class property name in hidden picker (In editItemInPopups mode)
+    #if("$!SuggestDisplay_itemsEditable" != "" &amp;&amp; $SuggestDisplay_itemsEditable == true)
+       ## New js/css calls
+       #set($discard = $xwiki.ssx.use("XWiki.SuggestDisplayEditItems"))
+       #set($discard = $xwiki.jsx.use("XWiki.SuggestDisplayEditItems"))
+       #set ($attachmentPickerDocName = 'XWiki.AttachmentSelector')
+       $xwiki.ssx.use($attachmentPickerDocName)
```

Comment:
Why don't you do `#set($discard = $xwiki.ssx.use($attachmentPickerDocName))` like one line above?